filebeat/crawler/crawler.go

```diff
@@ -16,7 +17,7 @@ type Crawler struct {
 	prospectorConfigs []*common.Config
 	out               prospector.Outlet
 	wg                sync.WaitGroup
-	reloader          *prospector.ProspectorReloader
+	reloader          *cfgfile.Reloader
```

Comment:
This is local to the crawler, so I think the naming fits as it reloads crawler specific configs (prospectors).